README.md

```diff
@@ -126,6 +126,13 @@ One may seed the cache using the following Rake Task:
 rake liberate:arks:seed_cache
 ```
 
+In development, when running traject command, set the `FIGGY_ARK_CACHE_PATH` to point to `spec/fixtures/marc_to_solr/figgy_ark_cache` in the local environment. e.g. 
+```bash
+export FIGGY_ARK_CACHE_PATH=spec/fixtures/marc_to_solr/figgy_ark_cache
+bundle exec traject -c marc_to_solr/lib/traject_config.rb pathToFile -u http://localhost:solr-port/solr/marc-liberation-core-development
```

Comment:
@maxkadel I dont think we need the traject command to index records in the section Seeding the cache. I just viewed the full file and realized that its in this section. Can we remove lines 132 and 133?